CHANGELOG.md

```diff
@@ -7,13 +7,21 @@
 [//]: # " ### Bug Fixes"
 [//]: # " - Fix the bug."
 
+## Development 0.1.0dev (to be released)
+
+### Improvements
+
+- Add to the documentation instructions for maintainers to make a new release. (@nathanshammah, gh-#332)
```

Comment:
There are probably more changes here, no? E.g. pyQuil support, `instack` and `outstack` are now private attributes of factories, `PolyFactory.static_reduce` has been removed, others?

Reply:
@rmlarose This is only for the release. I am opening a new PR on this, putting in all the others' PRs going through the merged ones since the latest 0.1a2 version.